firebase-firestore/CHANGELOG.md

```diff
@@ -1,10 +1,18 @@
-# Unreleased
-- [changed] Increases the aggressiveness of network retires when an app's
+# 23.0.3
+- [fixed] Fixed an issue when loading a data bundle with
+  multi-bytes unicode character leads to failures.
```

Comment:
nit: "multi-bytes unicode character" -> "multi-byte Unicode characters". The "U" in "Unicode" should be uppercase and the plural needs to be tweaked.

Reply:
Done.